go/vt/sqlparser/ast.go

```diff
@@ -86,6 +85,12 @@ type (
 		SQLNode
 	}
 
+	// Explain is an interface that represents the Explain statements
+	Explain interface {
+		iExplain()
+		Statement
```

Comment:
Nitpick: first define `Statement`, the interface that we extend, than add the functions this `interface` implements.